snapcraft/internal/repo/_platform.py

```diff
@@ -0,0 +1,36 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import platform
+
+from ._deb import Ubuntu
+
+_DEB_BASED_PLATFORM = [
+    'Ubuntu',
+    'Debian',
+]
+
+
+def _is_deb_based():
+    return platform.linux_distribution()[0] in _DEB_BASED_PLATFORM
```

Comment:
Forewarning here, `platform.linux_distribution()` is going away in Python 3.7. Use [`distro`](https://pypi.python.org/pypi/distro). You don't need to address this now if you don't want to, but it's a good time to warn about this.